### DISCUSSION TITLE
✨ [Feature]: 完善Modal组件功能

### DISCUSSION THREAD

**Discussion by kagol** [DevCloudFE/vue-devui#1391] [General]
### What problem does this feature solve

Modal组件还有以下功能未完成：
- [x] 标准对话框
- [x] 自定义对话框
- [x] 拦截对话框关闭
- [ ] 信息提示
- [ ] 警告弹出框
- [ ] 弹出框按钮状态
- [ ] 自动聚焦
- [ ] 自定义内容模板
- [ ] 避免滚动和抖动

参考：
https://devui.design/components/zh-cn/modal/demo

### What does the proposed API look like

参考：
https://devui.design/components/zh-cn/modal/demo

**Comment by sakanaovo** [comment]
认领
